### DISCUSSION TITLE
Hello others can see my profile

### DISCUSSION THREAD

**Discussion by darknebular** [community/community#63455] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I would to know why my profile is not seeing for others?
https://github.com/darknebular/Wrapper_VideoStation/

**Comment by cehfisher** [comment]
Hi @darknebular - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
